tool/teleport/common/teleport.go

```diff
@@ -318,6 +318,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	dumpNodeConfigure.Flag("token", "Invitation token to register with an auth server.").StringVar(&dumpFlags.AuthToken)
 	dumpNodeConfigure.Flag("auth-server", "Address of the auth server.").StringVar(&dumpFlags.AuthServer)
 	dumpNodeConfigure.Flag("labels", "Comma-separated list of labels to add to newly created nodes ex) env=staging,cloud=aws.").StringVar(&dumpFlags.NodeLabels)
+	dumpNodeConfigure.Flag("join-method", "Method to use to join the cluster").EnumVar(&dumpFlags.JoinMethod, "token", "iam", "ec2")
```

Comment:
```suggestion
	dumpNodeConfigure.Flag("join-method", "Method to use to join the cluster. One of: token, iam, or ec2").EnumVar(&dumpFlags.JoinMethod, "token", "iam", "ec2")
```

(unless EnumVar will already dump the options in the help?)

Reply:
afaik `EnumVar` only dumps the options once you get it wrong, in other places we tend to add the hint to the help string like `(token, ec2, iam)`